react/features/filmstrip/subscriber.web.js

```diff
@@ -73,12 +74,12 @@ StateListenerRegistry.register(
     });
 
 /**
- * Listens for changes in the chat state to calculate the dimensions of the tile view grid and the tiles.
+ * Listens for changes in the chat state to recompute available width.
  */
 StateListenerRegistry.register(
     /* selector */ state => state['features/chat'].isOpen,
     /* listener */ (isChatOpen, store) => {
-        const state = store.getState();
+        const { innerWidth, innerHeight } = window;
```

Comment:
This won't work on mobile.

Reply:
But I was under the impression this only executes on web since it's in `subscriber.web.js`

Reply:
Doh, my bad, indeed it will only work on web. Nothing to see here, move along!